### DISCUSSION TITLE
Template VaultWarden

### DISCUSSION THREAD

**Discussion by Silbad** [Dokploy/dokploy#554] [Ideas]
Hello,
A new great adding : https://github.com/dani-garcia/vaultwarden 
Unofficial Bitwarden compatible server written in Rust, formerly known as bitwarden_rs 
Doc : https://github.com/dani-garcia/vaultwarden/wiki/Using-Docker-Compose
Thanks

**Comment by nktnet1** [comment]
Resolved by
- https://github.com/Dokploy/dokploy/pull/613
